.github/workflows/close-release-issue.yml

```diff
@@ -0,0 +1,30 @@
+name: close release issue
+
+on:
+  workflow_run:
+    workflows: [release]
```

Comment:
We can also configure the workflow to run after a release is created instead of waiting for the `release` workflow to be completed